apps/style/courses.scss

```diff
@@ -0,0 +1,56 @@
+@import 'color';
+@import 'aniGif';
+
+@media print {
+  /* we have many components that we never want to include in a print view */
+  #environment_tag,
+  #uitest-version-selector,
+  .announcement-notification,
+  .course-overview-top-row,
+  .teacher-info-box,
+  .uitest-go-to-unit-button,
+  .extra-links,
+  .footer,
+  .header-wrapper,
+  .push {
+    display: none !important;
+  }
```

Comment:
how many of these actually need `!important`? If there's any that don't, we should break this up into two different styles, to minimize our use of that flag wherever possible